### DISCUSSION TITLE
Option to disable auto approve for admin and owner accounts

### DISCUSSION THREAD

**Discussion by RaptorCentauri** [seerr-team/seerr#190] [Ideas]
I would like the option to turn off auto approve for admin/ owner accounts. Sometimes my request list is larger than I have capacity for, and I would like to manually decide what and when to approve for download.

**Comment by ghost** [comment]
I agree on this idea, but for a different reason. Woke up to 200+ random requests due to a weak password on the owner account.

**Comment by lephtHanded** [comment]
+1 for this.

**Comment by gauthier-th** [comment]
You can use the watchlist feature to "bookmark" some items you want to accept later.

**Comment by MaximumFish** [comment]
Adding my support for this. I'd like to use the Plex watchlist feature to auto-request things, but due to my (admin) user being the first in the list when loading up Plex, my wife tends to default to using that one instead of her own.

**Comment by chazzawalla** [comment]
+1, I tend to request media on any little enormity might be by good. Would be nice if they’d just get queued so I can research them more later before actually approving 

The option is right there in the settings….locked behind grey text…

**Reply by Motz31** [reply]
> +1, I tend to request media on any little enormity might be by good. Would be nice if they’d just get queued so I can research them more later before actually approving
> 
> The option is right there in the settings….locked behind grey text…

+1, I do it the same way
Right now I have to use 2 separate useres to achive the same. One can only request and the other one needs to approve the requests.

**Comment by Kaarug** [comment]
So many different reasons for the exact same change. Just let us remove (and re-add) the permissions as admins for admins. ...

**Comment by Tatsu941** [comment]
It would also make apps like silo work way better with seerr since you can acept requests via API but not have them auto approved

**Comment by KLEPTOROTH** [comment]
+1 from me, I want to use it like a watchlist and only approve when I am ready to watch
